OutOfSchool/OutOfSchool.DataAccess/Models/Configurations/AverageRatingConfiguration.cs

```diff
@@ -0,0 +1,14 @@
+﻿
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace OutOfSchool.Services.Models.Configurations;
+
+internal class AverageRatingConfiguration : IEntityTypeConfiguration<AverageRating>
+{
+    public void Configure(EntityTypeBuilder<AverageRating> builder)
+    {
+        builder.HasIndex(x => x.EntityId);
+        builder.Property<bool>("IsDeleted").HasDefaultValue(false);
```

Comment:
We really need this default value?

Reply:
Yes, because the app raises an exception when we try to store data in the table. It was the same situation when I was developing the region admin. I also used the default value for this property.

Reply:
Ok. Lets move this line to ModelBuilderExtension.ApplySoftDelete and apply it to all deleting entities as default.

Reply:
I've done it.